test/integration/AppBar/AppBar.spec.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env mocha */
+import React from 'react';
+import {mount} from 'enzyme';
+import {assert} from 'chai';
+import AppBar from 'src/AppBar';
+import IconButton from 'src/IconButton';
+import FontIcon from 'src/FontIcon';
+import getMuiTheme from 'src/styles/getMuiTheme';
+
+describe('<AppBar />', () => {
+  const muiTheme = getMuiTheme();
+  const mountWithContext = (node) => mount(node, {context: {muiTheme}});
+
+  it('renders a FontIcon inside the iconElementLeft with custom color', () => {
+    const wrapper = mountWithContext(
+      <AppBar
+        iconElementLeft={
+          <IconButton>
+            <FontIcon className="material-icons" color="red">home</FontIcon>
+          </IconButton>
+        }
+      />
+    );
+
+    assert.strictEqual(wrapper.find(FontIcon).node.props.color, 'red', 'FontIcon should have color set to red');
+    assert.strictEqual(wrapper.find(FontIcon).node.props.style.color, undefined,
```

Comment:
@andrejunges can we make function call args line breaking consistent. So one of two options:

1. All on one line (if they fit)
2. One per line

Mixing it up makes it hard to read